nilearn/glm/first_level/first_level.py

```diff
@@ -1015,6 +1017,86 @@ def first_level_from_bids(
     The subject label of the model will be determined directly
     from the :term:`BIDS` dataset.
 
+    kwargs: :obj:`dict`
+
+        .. added:: 0.11.0
+
+    Keyword arguments to be passed to functions called within this function.
+
+    Kwargs prefixed with ``confound_``
+    will be passed to :func:`~nilearn.interfaces.fmriprep.load_confounds`.
+    This allows to ``first_level_from_bids`` to return
+    a specific set of confounds by relying confound loading strategies
+    defined in :func:`~nilearn.interfaces.fmriprep.load_confounds`.
+    If no kwargs are passed, ``first_level_from_bids`` will return
+    all the confounds available in the confounds TSV files.
```

Comment:
@bthirion let me know if this helps clarify how to use this.

I prefer to add examples here and refer users to the doc of load_confounds for the details to avoid doc duplications.

Reply:
Agreed !